### PR TITLE
Adds styling to homepage

### PR DESCRIPTION
**Pull request by AJCJ1**
See comments

### CODE REVIEW

File: app/assets/stylesheets/components/_card_home.scss [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
-.cards-container-custom {
-  display: grid;
-  width:100vh;
-  grid-template-columns:1fr 1fr;
-  grid-gap: 16px;
-  margin:auto;
+.card-img-cust {
+  object-fit: cover;
+  object-position: center;
+  height:20vh;
```
Comment (AJCJ1) [inline app/assets/stylesheets/components/_card_home.scss:15]: cards now show 2 per row, neatly and re-styled.

File: app/assets/stylesheets/components/_carousel_arrows.scss [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+@keyframes flash {
+  from {background-color: transparent;}
+  to {background-color: orange;}
+}
+
+.arrow-cust:hover {
+  animation-name: flash;
+  animation-duration: 1s;
+  animation-iteration-count: 15;
+}
```
Comment (AJCJ1) [inline app/assets/stylesheets/components/_carousel_arrows.scss:11]: when you hover over the arrows in carousel, runs a cheesy animation like car indicators.

File: app/assets/stylesheets/components/_index.scss [hunk 3]
```diff
@@ -6,3 +6,4 @@
 @import "card_home";
 @import "banner";
 @import "show_banner";
+@import "carousel_arrows";
```
Comment (AJCJ1) [inline app/assets/stylesheets/components/_index.scss:9]: adds specific component styling to index

File: app/assets/stylesheets/pages/_index.scss [hunk 4]
```diff
@@ -1,3 +1,43 @@
...
+  width:100;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  row-gap: 32px;
+  align-items: center;
+  justify-items: center;
+}
+
+
+.header-custom {
```
Comment (AJCJ1) [inline app/assets/stylesheets/pages/_index.scss:16]: styling for homepage banner

File: app/views/cars/index.html.erb [hunk 5]
```diff
@@ -1,46 +1,45 @@
...
+            <% end %>
           <% end %>
-          <% end %>
-      </div>
+        </div>
+    <button class="carousel-control-prev arrow-cust" type="button" data-bs-target="#carousel-car-card-<%=car.id%>" data-bs-slide="prev">
+      <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+      <span class="visually-hidden">Previous</span>
+    </button>
+    <button class="carousel-control-next arrow-cust" type="button" data-bs-target="#carousel-car-card-<%=car.id%>" data-bs-slide="next">
```
Comment (AJCJ1) [inline app/views/cars/index.html.erb:26]: carousel replaced with better, more functional one.

File: app/views/shared/_navbar.html.erb [hunk 6]
```diff
@@ -3,7 +3,7 @@
...
...vg" height="48" width="48"><path d="M13.8 40q-2.25 0-3.975-1.675T7.95 34.25Q6.1 33.1 5.075 31.325T4.05 27.45q0-2.6 1.525-4.675Q7.1 20.7 9.6 20.15L5.55 15.9l-.95 1q-.45.5-1.075.5t-1.075-.5Q2 16.45 2 15.85q0-.6.45-1.1l4.05-4.2q.45-.45 1.075-.45t1.075.45q.45.5.45 1.1 0 .6-.45 1.05l-.95 1.05 3.6 3.85 1.95-5.85q.55-1.65 1.925-2.7Q16.55 8 18.25 8h11.6q1.7 0 3.1 1.05 1.4 1.05 1.9 2.7l2.5 8.3q2.85.25 4.75 2.325T44 27.45q0 2.1-1.025 3.875T40.1 34.25q-.15 2.4-1.875 4.075Q36.5 40 34.25 40q-2.1 0-3.675-1.45Q29 37.1 28.6 35h-9.15q-.4 2.1-1.975 3.55Q15.9 40 13.8 40Zm-.1-20.05h8.75V11h-4.2q-.75 0-1.3.475t-.8 1.175Zm11.75 0h8.75l-2.25-7.3q-.25-.7-.8-1.175Q30.6 11 29.85 11h-4.4Zm-6 12.05h9.15q.55-1.8 2.15-2.9 1.6-1.1 3.5-1.1 1.6 0 2.95.85t2.2 2.2q.85-.65 1.225-1.625Q41 28.45 41 27.45q0-1.9-1.3-3.2-1.3-1.3-3.2-1.3H11.55q-1.9 0-3.2 1.3-1.3 1.3-1.3 3.2 0 1 .4 1.975.4.975 1.2 1.625.8-1.45 2.175-2.25Q12.2 28 13.8 28q1.9 0 3.5 1.1t2.15 2.9Zm-5.65 5q1.3 0 2.15-.875.85-.875.85-2.125 0-1.3-.85-2.15Q15.1 31 13.8 31q-1.25 0-2.125.85T10.8 34q0 1.25.875 2.125T13.8 37Zm20.45 0q1.3 0 2.15-.875.85-.875.85-2.125 0-1.3-.85-2.15-.85-.85-2.15-.85-1.25 0-2.125.85T31.25 34q0 1.25.875 2.125T34.25 37Zm-10.2-9.55Z"/></svg>
```
Comment (AJCJ1) [inline app/views/shared/_navbar.html.erb:6]: no need for class color white when style is fill #FFFF

File: app/views/cars/show.html.erb [hunk 7]
```diff
@@ -12,11 +12,11 @@
           <% end %>
           <% end %>
       </div>
-    <button class="carousel-control-prev" type="button" data-bs-target="#carousel-car-card" data-bs-slide="prev">
+    <button class="carousel-control-prev arrow-cust" type="button" data-bs-target="#carousel-car-card" data-bs-slide="prev">
```
Comment (AJCJ1) [inline app/views/cars/show.html.erb:15]: adds custom class for custom styles

File: app/views/cars/index.html.erb [hunk 8]
```diff
@@ -1,46 +1,45 @@
...
+      </div>
+      <div class="card-body border-top">
+        <%= link_to "See more", car_path(car), class:"m-2 text-decoration-none" %>
+      </div>
 
-<% end %>
+    </div> <%# end of individual card container %>
+  <% end %> <%# end of @cars.all statement %>
 
-<p><%= link_to 'Add a car', href='cars/new', class:"btb btn-primary"%></p>
```
Comment (AJCJ1) [inline app/views/cars/index.html.erb:46]: placed in banner

File: app/assets/stylesheets/components/_carousel_arrows.scss (hunk 2 above)
Comment (vdequercize) [reply app/assets/stylesheets/components/_carousel_arrows.scss:11]: Cant' wait to see it !

**Review by vdequercize — APPROVED** [review]

**Pull request merged**